react/Viewer/Readme.md

```diff
@@ -185,3 +185,8 @@ And then configure the [webpack worker-loader](https://github.com/webpack-contri
 ### Only works with React
 
 The `Viewer` can be used only in a `React` Application. You can't use it with `Preact`.
+
+### Special case: client-side encrypted files
+
+When a file is client-side encrypted, it is not possible to get the download link from the server anymore. 
+Hence, an additional `currentURL` prop must be given to the `<Viewer>`, that redirects to the decrypted file. 
```

Comment:
inside the Viewer/index.jsx, I would add a 

if(isEncrypted(File) && !currentURL) { 
  logger.error('big error to sentry')
}

Reply:
Actually, the absence of `currentURL` is not necessarily an error: the file will be decrypted asynchronously by the app and when it's done, pass it to the viewer

Reply:
> Hence, an additional `currentURL` prop must be given to the `<Viewer>`, that redirects to the decrypted file. 

`must` ?

> or Actually, the absence of currentURL is not necessarily an error

either the documentation should be updated or I would stay with the idea of a check inside the component.

ps: available  for a call, to end these long discussions  in the  review 🙏

Reply:
I added some clarifications in the doc : [4bcb582](https://github.com/cozy/cozy-ui/pull/2112/commits/4bcb582d80fb8353df90e0a13478ac0db4d24381)